packages/server/api/controllers/reviews.controller.js

```diff
@@ -101,14 +101,13 @@ const addReview = async (movieId, userId, rating, reviewText) => {
   }
 
   try {
-    const review = await knex('reviews').insert({
+    await knex('reviews').insert({
       movie_id: movieId,
       user_id: userId,
       rating,
       review_text: reviewText,
       created_at: moment().format(),
     });
-    return review;
   } catch (error) {
     throw new HttpError(error.message, 500);
   }
```

Comment:
almost there! 

What I would suggest doing would be discarding try/catch at all here (you are caching in router anyway) and plainly do:
```
return knex('reviews').insert(....)
```
at the and of the function. Note I also omitted the await as the function is promise anyway :)

Let me know if that makes sense!
And please, if you are waiting for specifically my feedback re-request review from me :) It's easier for me to not get lost in notifications and reply faster :)

Reply:
yes it make sense, so removing the try/catch since we handle it in router and add return to handle the returned promise